src/shop/actions/discount/get-discount.action.ts

```diff
@@ -13,12 +14,18 @@ export const getDiscount = async (
       StringUtils.bindContext(endpoints.MANAGE_DISCOUNT, { id }),
     );
 
-    if (response.status !== 200) {
-      throw errorFactory.create(DiscountErr.FETCH_FAILED);
+    switch (response.status) {
+      case 200:
+        return response.data;
+      case 403:
+        throw errorFactory.create(DiscountErr.GET_DISCOUNTS_FORBIDDEN);
```

Comment:
```suggestion
        throw errorFactory.create(DiscountErr.GET_DISCOUNT_FORBIDDEN);
```

Pas nécessaire de mettre "discount" au pluriel puisqu'on ne récupére qu'une remise ici.